pkg/clone/retry.go

```diff
@@ -24,7 +24,7 @@ func Retry(ctx context.Context, options RetryOptions, f func(context.Context) er
 	start := time.Now()
 	retries := 0
 	b := backoff.WithContext(backoff.WithMaxRetries(IndefiniteExponentialBackOff(), options.MaxRetries), ctx)
-	err := backoff.RetryNotify(func() (err error) {
+	operation := func() (err error) {
```

Comment:
This anonymous function is used as the first argument to function call `backoff.RetryNotify()`. I thought it's a bit clearer to give it a name